sql/api/src/main/scala/org/apache/spark/sql/errors/QueryParsingErrors.scala

```diff
@@ -621,9 +621,8 @@ private[sql] object QueryParsingErrors extends DataTypeErrorsBase {
   def unsupportedFunctionNameError(funcName: Seq[String], ctx: ParserRuleContext): Throwable = {
     new ParseException(
       errorClass = "INVALID_SQL_SYNTAX.MULTI_PART_NAME",
-      messageParameters = Map(
-        "statement" -> toSQLStmt("CREATE TEMPORARY FUNCTION"),
-        "funcName" -> toSQLId(funcName)),
+      messageParameters =
+        Map("statement" -> toSQLStmt("CREATE TEMPORARY FUNCTION"), "name" -> toSQLId(funcName)),
```

Comment:
I see a lot of these changes (`funcName` to `name`) in this PR

is this related to the core changes? (supporting ident in SET CATALOG)

Reply:
Yes, it's not always function name now, but can also be catalog name.